CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [(Unreleased) hipSPARSE 1.10.4 for ROCm 4.1.0]
 ### Added
-- Generic API support, including Axpby, Gather, Scatter, Rot, SpVV, SpMV, SparseToDense, DenseToSparse and SpGEMM
+- Generic API support, including Axpby, Gather, Scatter, Rot, SpVV, SpMV, SpMM, SparseToDense, DenseToSparse and SpGEMM
```

Comment:
I think thats going to be part of 4.2. 4.1 branching already happened a while ago.

Reply:
Fixed

Reply:
I added the "unreleased" lines for rocm 4.2 with spmm. This can be updated with whatever other features in addition to spmm end up going in 4.2 and the hipsparse version can be updated by whoever ends up push hipsparse to master.